Validation/RecoVertex/python/pvRecoSequence_cff.py

```diff
@@ -61,22 +63,27 @@
         minSiliconLayersWithHits = cms.int32(5),
         maxD0Significance = cms.double(5.0),
         minPt = cms.double(0.0),
+        maxEta = cms.double(5.0),
         trackQuality = cms.string("any")
     ),
 
     TkClusParameters = cms.PSet(
         algorithm   = cms.string("DA"),
         TkDAClusParameters = cms.PSet(
             coolingFactor = cms.double(0.6),  #  moderate annealing speed
-            Tmin = cms.double(4.),            #  end of annealing
+            Tmin = cms.double(2.4),            #  end of annealing
+            Tpurge = cms.double(2.),            #  end of annealing
+            Tstop = cms.double(0.5),
             vertexSize = cms.double(0.01),    #  
             d0CutOff = cms.double(3.),        # downweight high IP tracks
-            dzCutOff = cms.double(4.)         # outlier rejection after freeze-out (T<Tmin)
+            dzCutOff = cms.double(4.),         # outlier rejection after freeze-out (T<Tmin)
+            uniquetrkweight = cms.double(0.9)
```

Comment:
I suspect these changes are not needed as the algorithm is `DA` and not `DA_vect`, and the parameters were added only for the latter.